DokuwikiPage.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * This is the Dokuwiki export for FINDOLOGIC.
+ *
+ * If any bugs occur, please submit a new issue
+ * @see https://github.com/findologic/dokuwiki-plugin-findologic-xml-export/issues/new
+ * @author Dominik Brader <support@findologic.com>
+ */
+
+class DokuwikiPage
+{
+    /**
+     * This is the time format as it gets outputted to the template.
+     * DAY.MONTH YEAR (HOUR:MINUTE)
+     */
+    const TIME_FORMAT = '%d.%B %Y (%H:%M)';
+
+    /**
+     * @var string ID of the DokuWiki page.
+     */
+    public $id;
+
+    /**
+     * @var string URL of the DokuWiki page (absolute).
+     */
+    public $url;
+
+    /**
+     * @var string Author of the last change that was made to the DokuWiki page.
+     */
+    public $author;
+
+    /**
+     * @var string Formatted timestamp of last edited timestamp.
+     */
+    public $lastEdit;
```

Comment:
Save time as `DateTime` object, this provides type-safety, object-oriented coding and can be properly compared to other `DateTime`s.

Reply:
Note4Me: should be done locally